packages/shared/babel-ast-utils/src/index.js

```diff
@@ -60,6 +60,11 @@ export async function generate({
   if (generated.rawMappings) {
     map = new SourceMap();
     map.addIndexedMappings(generated.rawMappings);
+
+    let originalMap = await asset.getMapBuffer();
+    if (originalMap && map) {
+      map.extends(originalMap);
```

Comment:
This fixes the issue @wbinnssmith discovered in Kitchen Sink.

Ideally we would let Babel handle this by loading it into the AST in the beginning of a transform, although I haven't seen a way to do this without using `@babel/core` directly :(

But it seems to work surprisingly well so it's probably not that big of an issue.

Reply:
I agree, but why did this occur in the example? Because the babel transformer runs twice? 

(the ` && map` seems unnecessary)

Reply:
Probably although in that case shouldn't it be reusing the ast?

It will always be neccesary though as babel can run after coffeescript or something and would need to do the same thing...

And you're right `&& map` is not required, it used to be outside the if before I refactored and forgot to change that...